protocol/0026-AUCT-auctions.md

```diff
@@ -6,7 +6,8 @@ Auctions are a trading mode that 'collect' orders during an *auction call period
 
 In comparison to continuous trading, the auction mode for a market, is a state of the orderbook where each order placed is just sitting on the book, for a given period of time or until some requirements are met (called `call period`), then the matching orders are uncrossed.
 
-They are mostly useful in less liquid markets, or in specific scenarios where a price must be determined, i.e. at opening of a market, when a potentially excessively large price move might occur (price monitoring) or when liquidity needs to be sourced and aggregated (liquidity monitoring). In traditional markets (where markets open and close every day) we can run an open and closing auction for the price to stabilise at both ends.
+They are mostly useful in less liquid markets, or in specific scenarios where a price must be determined, i.e. at opening of a market, when a potentially excessively large price move might occur (price monitoring).
```

Comment:
Just a point of clarification: should we still have some form of liquidity monitoring during the opening auction period (ie extend the opening auction period if no liquidity for the market has been provided)?

Reply:
Nope, because any LP who commits during the opening auction can then remove the liquidity (in the SLA world they'll just get some small penalty). So let's not bother at all.